utils/nexus.go

```
@@ -75,7 +76,8 @@ func (n *NexusFileSharing) Download(fileName string, toDirectory string) error {
 		return fmt.Errorf(statusCodeError, res.StatusCode())
 	}
 
-	return UntarGZippedStream(toDirectory, bytes.NewBuffer(res.Body()))
+	defer func() { _ = res.RawBody().Close() }()
```

if you ignore error ) you can write like that  `defer res.RawBody().Close()`

---

if I will do it in this way, "golangci-lint" returns an error, so I did it in this way.

---

if we go this way, then let's not ignore the error, please